hack/chains/rekor-verify-taskrun.sh

```diff
@@ -32,24 +32,34 @@ echo $TRANSPARENCY_URL
 pause
 
 title "Take a look at it"
-curl-json $TRANSPARENCY_URL | yq e . -P -
+curl-json $TRANSPARENCY_URL | yq-pretty
 pause
 
+# We could use rekor-cli here and it should be the same, e.g:
+# `rekor-cli get --log-index $LOG_INDEX --format json | jq ...`
+#
+BODY_DATA=$( curl-json $TRANSPARENCY_URL | jq -r 'values[].body' )
+ATTESTATION_DATA=$( curl-json $TRANSPARENCY_URL | jq -r 'values[].attestation' )
+
 title "Extract the rekor body"
-curl -s -H "Accept: application/json" $TRANSPARENCY_URL | jq -r 'values[].body' | base64 -d | yq e . -PC -
+echo "$BODY_DATA" | base64 -d | yq-pretty
 pause
 
-# Comment this out because there is no attestation data in the kaniko build task
-#title "Extract the rekor attestation"
-#curl -s -H "Accept: application/json" $TRANSPARENCY_URL | jq -r 'values[].attestation.data' | base64 -d | base64 -d | yq e . -PC -
-#pause
+if [[ $ATTESTATION_DATA = '{}' ]]; then
+  title "No attestation found"
+else
+  title "Extract the rekor attestation"
+  # It really is base64 encoded twice here
+  echo $ATTESTATION_DATA | jq -r .data | base64 -d | base64 -d | yq-pretty
```

Comment:
fyi `jq` itself can decode base64: `echo $ATTESTATION_DATA | jq -r '.data | @base64d | @base64d' | yq-pretty`

Reply:
That's neat! I didn't know that. I don't want to change it here, but maybe later we could use that trick in a few places to save some processes.